spreadsheets/collision_rank.yaml

```diff
@@ -0,0 +1,943 @@
+# reserved for UX/UI
+- _reserved: {from: 1, to: 100}
```

Comment:
Can just be 1 to 99

Reply:
(This and the next few suggestions will make documentation cleaner.)

Reply:
I changed the ranges to "round numbers" in 82d74dc.